draft-ietf-rats-corim.md

```diff
@@ -1868,10 +1868,10 @@ Otherwise, `cryptokeys` does not match.
 
 ##### Comparison for Integrity Registers {#sec-cmp-integrity-registers}
 
-For each Integrity Register entry in the Reference Value, the Verifier will use the associated identifier to look up the matching Integrity Register entry in Evidence.
+For each Integrity Register entry in the Reference Value, the Verifier will use the associated identifier (i.e., `integrity-register-id-type-choice`) to look up the matching Integrity Register entry in Evidence.
 If no entry is found, the Reference Value does not match.
-Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}}.
-Note that it is not required for all the entries in Evidence to be used during matching: the Reference Value could consist of a "quote" (in TPM parlance) of just a subset of the register space.
+Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after the cross-product equivalence has been found according to {{sec-comid-integrity-registers}}.
```

Comment:
Is it possible to simplify this language, before we merge?

Reply:
is it necessary? Today is cut-off day. And I am okay enough with the proposal

Reply:
I find there is some disconnect in the statement!

the digest comparison proceeds as defined in {{}} after the cross-product equivalence of what?

Do you want to say a set of entries or what ?

Reply:
Section {{sec-comid-integrity-registers}} describes the process. I didn't want to restate it here. Maybe just remove "cross-product"?

Reply:
Works for me! Creates confusion otherwise!